### DISCUSSION TITLE
Can't start Sunshine after update. (Windows 11)

### DISCUSSION THREAD

**Discussion by MrDrumble** [LizardByte/community#462] [Sunshine] [answered]
I had Sunshine installed and working great for a month or so. Upgraded recently and went through the uninstall/reinstall process as prompted. Once it was finished, I went to start Sunshine and the shortcut in the start menu was gone. So I dug into Program Files to start it there, but I got the console message:

`To relaunch Sunshine successfully, use the shortcut in the Start Menu. Don't run Sunshine.exe manually`

I'm assuming there was some added command flags on the Start Menu shortcut, because as expected just making my own gave me the same error message. I looked at the bat files but nothing there was helpful either.

**Comment by ReenigneArcher** [comment]
I would suggest completely uninstalling Sunshine from program files, and then re-installing. Be sure to use the installer, not the portable version.

**Reply by MrDrumble** [reply]
Hmm, same result. Program was opened automatically and ran fine after the re-install. But after closing it, I get the same error when trying to run it manually.

**Reply by MrDrumble** [reply, marked as answer]
OK, tried another uninstall/reinstall. Deleted an existing shortcut (that gave the above error) from start menu before reinstalling.  Also left "run on startup" checked, which I don't want.

Working fine now, a new start shortcut was auto-created and it works.  Not sure which of those things fixed it.
